lib/reply.js

```diff
@@ -196,11 +207,15 @@ function sendStream (payload, res, reply) {
     }
   })
 
+  for (var key in reply._headers) {
+    res.setHeader(key, reply._headers[key])
+  }
```

Comment:
Why not `res.writeHead`?

Reply:
I’ll add a comment. We want to return 500s if the streams error before trying to send any data. That is the only way.

Reply:
Good point. I'm 👍  for the comment